packages/kotti-ui/source/kotti-field-inline-edit/KtFieldInlineEdit.vue

```diff
@@ -0,0 +1,356 @@
+<template>
+	<KtField
+		v-bind="{ field }"
+		ref="fieldRef"
+		:class="rootClasses"
+		:helpTextSlot="$slots.helpText"
+	>
+		<EditIcon v-if="!isEditing" />
+		<textarea
```

Comment:
Could be `<component :is="isMultiline ? 'textarea' : 'input'>`

Reply:
upon reading vue2 docs and running into the issue

using `component` requires the use of `.prop` modifier when updating the `value` for the HTML attribute to actually update

Because we don't want to support this hacky behavior, I kept the textarea and input 

upon migrating to vue3 where the issue doesn't seem (according to docs) to still exist, we can use a component here